base/test-core/src/test/java/org/openscience/cdk/config/AtomTypeFactoryTest.java

```diff
@@ -285,7 +285,7 @@ public void testGetAtomTypeFromMM2() throws Exception {
         Assert.assertEquals("S-[2];[H]{0,3}+=C.*+", atomType.getProperty(CDKConstants.SPHERICAL_MATCHER));
         Assert.assertEquals(Hybridization.SP2, atomType.getHybridization());
         Assert.assertTrue(atomType.getFlag(CDKConstants.IS_HYDROGENBOND_ACCEPTOR));
-        Assert.assertEquals(5, atomType.getProperty(CDKConstants.PART_OF_RING_OF_SIZE));
+        Assert.assertEquals(5, (int)atomType.getProperty(CDKConstants.PART_OF_RING_OF_SIZE));
```

Comment:
What is the warning here? You should probably be doing this: 

```
Assert.assertEquals((Integer)5, atomType.<Integer>getProperty(CDKConstants.PART_OF_RING_OF_SIZE));
```
or 

```
Assert.assertThat(atomType.<Integer>getProperty(CDKConstants.PART_OF_RING_OF_SIZE),
                          CoreMatchers.is(5));
```

Reply:
I actually wanted the second, but it seems that requires an upgrade to JUnit5. The warning was that `assertEquals()` was ambiguous and the compiler did not know if it had to use `assertEquals(double, double)` or `assertEquals(Object, Object)`.

Reply:
Second does not need Junit5

Reply:
It gave a warning that `Assert.assertThat()` was deprecated and wanted to focus first getting it compile. At this moment I'm looking into a NPE fail in `XYZWriterTest`.

Reply:
Oh right yeah they changed it in the recent JUnit patch release. Change for the sake of change - sigh. It's now `Matcher.assertThat()` I think.

Reply:
```MatcherAssert.assertThat()``` even